src/use-format-block.hook.js

```diff
@@ -6,10 +6,44 @@ export function useFormatBlock() {
   return {
     formatBlock(value) {
       if (value === document.queryCommandValue('formatBlock')) {
-        performCommandWithValue('div')
+        if (!!window.chrome && isInHTMLList()) {
+          removeHeader()
+        }
+        else {
+          performCommandWithValue('div')
+        }
       } else {
         performCommandWithValue(value)
       }
     },
   }
+}
+
+function isInHTMLList() {
```

Comment:
Could this function be simplified to the following:

```js
function isInHTMLList() {
  return document.queryCommandState('insertOrderedList') || document.queryCommandState('insertUnorderedList')
}
```

Reply:
🤦‍♂ I hate it when I wrote something I think is super clever and then someone just nails it with a line like this...good call